core/internal/api/api.go

```diff
@@ -115,6 +124,14 @@ func New(opts BackendOptions) *Backend {
 		logger:      opts.Logger,
 		apiKey:      opts.APIKey,
 		rateLimiter: rate.NewLimiter(maxRequestsPerSecond, maxBurst),
+		rlTracker: NewRateLimitTracker(RateLimitTrackerParams{
+			MinPerSecond: minRateLimit,
+			MaxPerSecond: maxRequestsPerSecond,
+
+			// TODO: Allow changing these through settings.
+			Smoothing:              0.2,
+			MinRequestsForEstimate: 5,
```

Comment:
should we also move them to const for now?

Reply:
I didn't because they're named here and they're not reused (the two reasons to use a named constant). I'm open to it though if you think I should!

Reply:
i was thinking for quick readability, but if you only use it here, i'm fine to keep it as is. Since like you said they are clearly named here